src/routes/safe/components/Apps/communicator.ts

```diff
@@ -7,8 +7,11 @@ import {
   ErrorResponse,
   MessageFormatter,
   METHODS,
+  RequestId,
 } from '@gnosis.pm/safe-apps-sdk'
-import { SafeApp } from './types.d'
+import { logError } from 'src/logic/exceptions/CodedException'
+import ErrorCodes from 'src/logic/exceptions/registry'
```

Comment:
You can import the codes from the same module as `logError` like so: `import { logError, Errors }`.